documentation/src/main/asciidoc/titles/developing/stories.adoc

```diff
@@ -1,8 +1,100 @@
 //Reference user stories or topics.
 //Please review authoring guidelines in the Contributors Guide.
 
+include::{stories}/assembly_getting_started.adoc[leveloffset=+1]
 include::{stories}/assembly_configuring_maven_repository.adoc[leveloffset=+1]
-include::{stories}/assembly_developing.adoc[leveloffset=+1]
+
+//Cache configuration
+include::{stories}/assembly_cache_modes.adoc[leveloffset=+1]
+include::{stories}/assembly_clustered_caches.adoc[leveloffset=+2]
+include::{stories}/assembly_cache_configuration.adoc[leveloffset=+1]
+include::{stories}/assembly_configuring_statistics_jmx.adoc[leveloffset=+2]
+include::{stories}/assembly_configuring_data_container.adoc[leveloffset=+1]
+include::{stories}/assembly_configuring_persistent_storage.adoc[leveloffset=+1]
+include::{stories}/assembly_creating_custom_cache_stores.adoc[leveloffset=+2]
+include::{stories}/assembly_migrating_cache_stores.adoc[leveloffset=+2]
+include::{stories}/assembly_partition_handling.adoc[leveloffset=+1]
+
+//Transactions and locking
+include::{stories}/assembly_transactions.adoc[leveloffset=+1]
+include::{stories}/assembly_locking.adoc[leveloffset=+1]
+
+//Cross-site replication
```

Comment:
not sure if it makes sense to have cross-site docs at this point of the document. It mentions the JGroups configuration changes but the JGroups topic is at the bottom of the doc

Reply:
Gotcha. Maybe it does make more sense to keep cross-site and JGroups together. TBH I wasn't sure about that placement myself. I added cross-site replication here to try and keep all the cache configuration parts together, backups/transactions/encoding/memory/...